output/cmf.eslint.txt

```diff
@@ -16,5 +16,8 @@ The react/require-extension rule is deprecated. Please use the import/extensions
 /home/travis/build/Talend/ui/packages/cmf/src/sagaRouter/router.js
   45:2  warning  Unexpected constant condition  no-constant-condition
 
-✖ 4 problems (1 error, 3 warnings)
+/home/travis/build/Talend/ui/packages/cmf/src/settings.js
+  29:37  error  Expected parentheses around arrow function argument having a body with curly braces  arrow-parens
```

Comment:
We really should do something about this prettier/eslint issue

Reply:
yes sorry :/

Reply:
not your fault